install/kubernetes/istio-auth.yaml

```diff
@@ -652,9 +681,10 @@ spec:
       - name: discovery
         image: gcr.io/istio-testing/pilot:5acaa9e26e7d9b239cbc72da63f5f9a3dac5a54e
         imagePullPolicy: IfNotPresent
-        args: ["discovery", "-v", "2"]
+        args: ["discovery", "-v", "2", "--admission-service", "istio-pilot-external"]
```

Comment:
So this will be on even in non cluster-wide? Is it for validation?

Reply:
Yes. This shouldn't be a problem because the admission control service only self-registers when the dependent secret w/certs is created. That secret is not created by default due to GKE 1.7.x bug (see workaround script).